x-pack/test_serverless/shared/config.base.ts

```diff
@@ -49,6 +63,17 @@ export default async () => {
 
   return {
     servers,
+    dockerServers: defineDockerServersConfig({
+      registry: {
+        enabled: !!dockerRegistryPort,
+        image: dockerImage,
+        portInContainer: 8080,
+        port: dockerRegistryPort,
+        args: dockerArgs,
+        waitForLogLine: 'package manifests loaded',
+        waitForLogLineTimeoutMs: 60 * 2 * 10000, // 2 minutes
```

Comment:
⚠️ Same as https://github.com/elastic/kibana/pull/192064/files#r1746730436

Reply:
Good catch! Addressed in [c9c06](https://github.com/elastic/kibana/pull/192064/commits/c9c06037f294e72fffce8990a398fd1586f912f1).